cmd/alertmanager/main.go

```diff
@@ -216,8 +220,8 @@ func main() {
 		inhibitor.Stop()
 		disp.Stop()
 
-		inhibitor = NewInhibitor(alerts, conf.InhibitRules, marker)
-		disp = NewDispatcher(alerts, NewRoute(conf.Route, nil), build(conf.Receivers), marker)
+		inhibitor = inhibit.NewInhibitor(alerts, conf.InhibitRules, marker)
```

Comment:
Technically that pattern is reserved if `lowercase(type) == package_name`. But I like verbs for package names :( What a Zwickmuehle.